centaur/src/main/scala/centaur/test/standard/CentaurTestCase.scala

```diff
@@ -52,14 +52,6 @@ case class CentaurTestCase(workflow: Workflow,
   }
 
   def containsTag(tag: String): Boolean = testOptions.tags.contains(tag)
-
-  def reliesOnCallCachingMetadataVerification: Boolean = {
```

Comment:
I don't know what the implication of removing this is. Is this related to references like this?

http://github.com/broadinstitute/cromwell/blob/187c49defd1ef3c2df4133e6e9b9c92edfce79b4/centaur/src/main/resources/standardTestCases/bucket_name_with_no_trailing_slash.test#L11-L11

Reply:
Yes, this PR is reverting https://github.com/broadinstitute/cromwell/pull/5322. That previous PR quickly gave up on tests like the one you linked. Test like `bucket_name_with_no_trailing_slash` are verifying metadata specific to call-caching results.

The net result of this PR: restore bq-report-and-retrying for any test that verifies call-caching results-- after invalidating the failed call-caching entries from the Cromwell database.